pkgs/development/libraries/tremor/default.nix

```diff
@@ -1,12 +1,12 @@
-{ stdenv, fetchsvn, autoreconfHook, pkgconfig, libogg }:
+{ stdenv, fetchgit, autoreconfHook, pkgconfig, libogg }:
 
 stdenv.mkDerivation rec {
-  name = "tremor-svn-${src.rev}";
+  name = "tremor-git-${src.rev}";
```

Comment:
Why does this have "git"/"svn" in the name of the package?

Reply:
Because those aren't really releases. The idea is to signal that this is a package build from a VCS repo instead of a normal release.

The introduction of those predates my activities with NixOS so that is just what I read between the lines.

Reply:
That's already getting covered by the hash for the version, so I think we should take it out.

Reply:
Also, it seems like some people call this libtremor https://repology.org/metapackage/tremor/versions

Reply:
Perhaps `tremor-2018-03-16`, so that versions get a correct ordering in future.  That by itself should seem like a non-release, and the name would be independent of VCS switches.

Reply:
Actually, [our docs](https://nixos.org/nixpkgs/manual/#sec-package-naming) imply `tremor-unstable-2018-03-16`, so let's go for that.